src/NUnitFramework/framework/Internal/Builders/DefaultTestCaseBuilder.cs

```diff
@@ -141,7 +145,7 @@ public Test BuildFrom(IMethodInfo method, Test parentSuite)
                     tests.Add(test);
             }
 
-            return tests.Count > 0
+            return shouldParameterizeForFewerThanTwoTests || tests.Count > 0
```

Comment:
How about changing the variable to something like `originalBuildersCount`?

The name is confusing to me on several levels, especially the use of "Two" rather than "One".